### ISSUE TITLE
[DOC] DataProc A100 example fails and seems to need more information about setting the zone

### ISSUE THREAD

**Issue by andygrove** [NVIDIA/spark-rapids#6343]
## Report incorrect documentation

**Location of incorrect documentation**

https://nvidia.github.io/spark-rapids/docs/get-started/getting-started-gcp.html#create-a-dataproc-cluster-using-mig-with-a100s

**Describe the problems or issues found in the documentation**

I tried running the script in the docs, and it failed with:

```
INVALID_ARGUMENT: Auto zone placement currently only supports [n1-, custom-, e2-, n2-, c2-, n2d-, m1-, m2-] machine types (found 'a2-highgpu-1g'). Please explicitly select a zone and try again.
```

**Steps taken to verify documentation is incorrect**

The documentation states:

```
export REGION=[Your Preferred GCP Region]
```

So I had set it like this:

```
export REGION=us-central1
```

When using the machine type `a2-highgpu-1g` (as used in this script), it is necessary to specify the zone in addition to the region.

I tried this:

```
export REGION=us-central1-a
```

This failed with:

```
INVALID_ARGUMENT: Must specify a zone in GCE configuration when using 'regions/global'. To use auto zone placement, specify regions/<non-global-region> in request path, e.g. regions/us-central1
```

**Suggested fix for documentation**

I'm not sure what the fix is yet.

**Comment by sameerz** [comment]
Try setting `ZONE=us-central1-a` 

Details at https://cloud.google.com/compute/docs/regions-zones/

**Comment by sameerz** [comment]
@mengdong any idea why auto zone placement does not work for a2 instances?  

```
INVALID_ARGUMENT: Auto zone placement currently only supports [n1-, custom-, e2-, n2-, c2-, n2d-, m1-, m2-] machine types (found 'a2-highgpu-1g'). Please explicitly select a zone and try again.```

**Comment by mengdong** [comment]
ah, I think autozone require all zone in that region have a2 instance, however in n1-central1 region, only n1-central1-a and c has a2 instance.

**Comment by sameerz** [comment]
It appears this issue was addressed in PR #6421 .  The documentation was updated as follows: 

https://github.com/NVIDIA/spark-rapids/blame/branch-22.10/docs/get-started/getting-started-gcp.md#L97-L126
